src/com/company/redi_exercises/Char_Counter.java

```diff
@@ -0,0 +1,21 @@
+package com.company.redi_exercises;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Char_Counter {
+    public static void main(String[] args) {
+        String word = "hello world";
+        Map<Character, Integer> repeatsChar = new HashMap<>();
+
+        for (int i = 0; i < word.length(); i++) {
+            if (!repeatsChar.containsKey(word.charAt(i))) {
```

Comment:
you can simplify the solution with using Map `getOrDefault()` method instead of using `containsKey()` and `get()`